pocketimdb/movies/urls.py

```diff
@@ -3,4 +3,4 @@
 from .views import MovieViewSet
 
 movieRouter = routers.SimpleRouter()
-movieRouter.register(r'movies', MovieViewSet)
+movieRouter.register(r'movies', MovieViewSet, basename='Movie')
```

Comment:
cemu sluzi `basename` ?

Reply:
Basename se inace automatski generise na osnovu `queryset` polja u `ViewSet` klasi, ali ja sam obrisao `queryset` polje u toj klasi jer imam `get_queryset` metodu (to smo juce pricali da li moze ili ne moze da se obrise to polje kada imamo get_queryset).